app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/landing/LandingViewModel.kt

```diff
@@ -91,8 +103,15 @@ class LandingViewModel @Inject constructor(
             LandingAction.CreateAccountClick -> handleCreateAccountClicked()
             is LandingAction.DialogDismiss -> handleDialogDismiss()
             is LandingAction.RememberMeToggle -> handleRememberMeToggled(action)
-            is LandingAction.EmailInputChanged -> handleEmailInputUpdated(action)
+            is LandingAction.EmailInputChanged -> handleEmailInputUpdated(action.input)
```

Comment:
All the `handle...` functions should take an action. If you want to make a reusable function, then the individual `handle...` function should call that reusable function.

Reply:
<img src="https://media3.giphy.com/media/X8HbeXDF7nzaM/giphy.gif"/>